tensorflow/core/common_runtime/mkl_layout_pass.cc

```diff
@@ -4211,6 +4513,10 @@ Status MklLayoutRewritePass::Run(const GraphOptimizationPassOptions& options) {
     return OkStatus();
   }
 
+  if (options.session_options != nullptr) {
+    num_intra_threads_ =
+        options.session_options->config.intra_op_parallelism_threads();
+  }
   auto process_graph = [&](std::unique_ptr<Graph>* g) {
```

Comment:
Is there a default value set to `num_intra_threads_` when `options.session_options` is nullptr?

Reply:
Hi @mdfaijul, in the implementation of `MklLayoutRewritePass` we initialise `num_intra_threads_` to the value of `port::MaxParallelism()` which gives number of schedulable CPUs which should be default number that is used when intra operation parallelism options is not set manually by user.